core/arch/arm/plat-imx/conf.mk

```diff
@@ -37,12 +43,14 @@ mx6s-flavorlist = \
 	mx6solosabresd \
 	mx6solosabreauto \
 
-mx7-flavorlist = \
+mx7d-flavorlist = \
 	mx7dsabresd \
 	mx7dpico_mbl \
+	mx7dclsom \
+
+mx7s-flavorlist = \
```

Comment:
Actually defined 2 cores for i.MX7S does not hurt, so do we need differentiate 7D and 7S?

Reply:
We did the differentiation for imx6(s/d/q) SoCs. This is how I would do things to match what we previously did.
Anyway, it's a minor change, I can discard it.